operator/pkg/helmreconciler/reconciler.go

```diff
@@ -28,6 +36,12 @@ import (
 	"istio.io/pkg/log"
 )
 
+const (
+	pollTimeout    = 60 * time.Second
+	pollInterval   = 2 * time.Second
+	istioNameSpace = "istio-system"
```

Comment:
it's better to get namespace from iop instance, right?

Reply:
ok, done